client.go

```diff
@@ -1916,6 +1918,17 @@ func (f *File) readFromWithConcurrency(r io.Reader, concurrency int) (read int64
 // This method is preferred over calling Write multiple times
 // to maximise throughput for transferring the entire file,
 // especially over high-latency links.
+//
+// If client uses concurrent writes, given r needs to implement one of the interfaces:
+//
+//	Len()  int
+//	Size() int64
+//	Stat() (os.FileInfo, error)
+//
+// or be an instance of [io.LimitedReader] to determine the number of possible
+// concurrent requests. Otherwise, reads/writes are performed in nonparallel, sliced
+// in chunks with the max packet size. ReadFromWithConcurrency explicit call can
```

Comment:
[English usage] I think we need to say “are not performed in parallel”. Or more consistent with other usage in the package “are performed sequentially” or “are not performed concurrently”.

[over-specific] I’m also unsure we need to include the detail that it’s sliced up into chunks, because both implementations sequential and concurrent do this. By calling this out here, it implies that the concurrent implementation doesn’t.